sqlite-conf.sql

```diff
@@ -0,0 +1,21 @@
+DROP TABLE ftpctxt;
```

Comment:
Ideally the document changes would be in a separate PR from the schema-related changes, as they don't relate to each other.